app/src/components/AuthenticationPart.vue

```diff
@@ -167,6 +172,17 @@ const goBack = () => {
       >
         <span>Back to Login</span>
       </button>
+      <hr class="my-8" />
+      <p class="flex justify-center font-semibold text-text text-m">
```

Comment:
Does it have "text-m" parameter?

Reply:
XD it doesn't, it should be text-base instead.